pkg/cidata/cidata.go

```diff
@@ -413,6 +414,24 @@ func getBootCmds(p []limayaml.Provision) []BootCmds {
 	return bootCmds
 }
 
+// getTimezone returns the timezone of host, to be used in the VM.
+func getTimezone() string {
+	_, GMTOffset := time.Now().Zone()
+	if GMTOffset%3600 != 0 {
+		logrus.Warnf("Host timezone offset is not a multiple of 3600 seconds: %d, skip timezone propagation", GMTOffset)
+	}
+
+	GMTOffsetHours := GMTOffset / 3600
+	var timezone string
+	if GMTOffsetHours > 0 {
+		timezone = fmt.Sprintf("Etc/GMT-%d", GMTOffsetHours)
+	} else {
+		timezone = fmt.Sprintf("Etc/GMT+%d", GMTOffsetHours)
```

Comment:
Don't you have to fix the sign of the offset?

```suggestion
		timezone = fmt.Sprintf("Etc/GMT+%d", -GMTOffsetHours)
```

Reply:
It's not a bug, perhaps I should have put a comment to it, but the sign is reversed [by design](https://stackoverflow.com/a/53089164/11004152)... I did test it locally with Alpine & Ubuntu VM BTW...

Reply:
I suspect you did not test it with a negative offset. You would end up with something like `Etc/GMT+-5`, which does not exist. I assume you would want `Etc/GMT+5`.

But I still think you should use the timezone name from the host instead of cooking up a name based on the GMT offset.

Reply:
> I suspect you did not test it with a negative offset

That's true. I missed that part. Will try to find the host location timezone.